docs/wallet/how-to-join-arbitrum-and-see-your-tokens.mdx

```diff
@@ -0,0 +1,132 @@
+:::
+
+Please follow the steps to add Arbitrum One Mainnet:
+
+- Navigate to [Arbiscan.io](https://arbiscan.io/)
+- Scroll down to the bottom of the page and click on the button `Add Arbitrum One Network`
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/Arbitrum-One.png")} alt="Arbitrum One" />
+
+- Once you click the button you will see a MetaMask notification, now click on **Approve**.
+You will be directed to Arbitrum's Mainnet in the network dropdown list. You can now close the dialog.
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/approve_on_metamask.png")} />
+
+
+<hr/>
+
+:::tip
+Alternatively you can add Arbitrum One Mainnet by following the steps below:
+
+Visit: [Chainlist](https://chainlist.org/chain/42161) and then click on the button `Add to Metamask`.
+:::
+
+</TabItem>
+
+<TabItem value="Sepolia">
+
+Please follow the steps to add Arbitum one Testnet (also known as Sepolia Testnet):
+
+1. Make sure that you have successfully added Arbitrum One Mainnet to your Metamask.
+2. Open Metamask and look for the network selection tool located at the upper left corner.
+3. Enable the "Show Test Networks" option.
+4. From the dropdown list, select "Sepolia" test network.
+
+<img class="marginTopBottomNormal" src={useBaseUrl("/img/wallet/add_sepolia_testnet.png")} />
+
+:::tip 
+Alternatively you can add Arbitrum One Testnet by following the steps below:
+
+Visit the official [Arbitrum One Testnet](https://testnet.arbiscan.io/) and then click on the button `Add to Metamask`.
+
+or
+
+Visit the official Documentation [Arbitrum One Testnet](https://developer.offchainlabs.com/docs/developer_quickstart) and then click on the button `Add to Metamask`.
+:::
+</TabItem>
+
+</Tabs>
+
+
+
+<hr/>
+
+
+
+### Add WXM token in MetaMask
+
+For the WXM token to appear in the wallet UI, you will need to add it manually.
+Note that your wallet can still receive tokens that haven’t been imported.
+
+:::note IMPORTANT NOTE
+Please make sure you have already added the **Mumbai-Testnet** and have connected to it by following the guide above!
+:::
+
+1. First, you will need the following details of the WXM token in order to add it:
+    1. **Token Contract Address:** ``0xDce8e61d46091C2b963785437D02eE1e4F3F7B36``
+    1. **Token Symbol:** ``MXW``
+    1. **Token Decimal:** ``18``
+
+You can verify those details on [Sepolia Arbiscan](https://sepolia.arbiscan.io/address/0xDce8e61d46091C2b963785437D02eE1e4F3F7B36).
+
+
+2. On MetaMask and click **[Import tokens]**.
+
+<img src={useBaseUrl("/img/wallet/add_WXM_toMetamask.png")} />
+
+3. Paste in the WXM’s contract address and MetaMask should automatically fill in the rest of the details. If not, add them manually. To finish, click **[Add Custom Token]**.
+
+4. Click **[Import Tokens]**.
+
+**You have now successfully added the WXM token to your MetaMask wallet!**
+
+:::note
+The ticker symbol for WXM is temprorarily until mainet **MXW**.
```

Comment:
This should go at the top of this page to avoid confusion. Also 2 typos here, 
1. temprorarily -> termorarily
2. mainet -> mainnet